src/main/java/io/appform/dropwizard/sharding/dao/operations/SelectParam.java

```diff
@@ -0,0 +1,13 @@
+package io.appform.dropwizard.sharding.dao.operations;
+
+import lombok.Builder;
+import lombok.Data;
+import org.hibernate.criterion.DetachedCriteria;
+
+@Builder
+@Data
```

Comment:
use @Value

Reply:
Not using this so this can be intercepted and mutated in observers, if needed.